Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -237,6 +238,9 @@ static bool ShouldDisplayGameListItem(const GameListItem& item)
   if (!show_platform)
     return false;
 
+  if (item.GetPlatform() == DiscIO::Platform::WII_WAD && !IOS::ES::IsChannel(item.GetTitleID()))
+    return false;
```

Comment:
I think it would make more sense to put this check in ISOFile.cpp (return early from the GameListItem constructor without setting `m_Valid` to `true`). The function it's in right now is (in master) only used for checking if a game should be hidden because of the user's settings, not checking if a game is invalid. I suppose you would need to increment the game list cache version if you do this, though.